crypto/fipsmodule/modes/gcm.c

```diff
@@ -155,7 +155,29 @@ static size_t hw_gcm_encrypt(const uint8_t *in, uint8_t *out, size_t len,
   if (!len_blocks) {
     return 0;
   }
-  aes_gcm_enc_kernel(in, len_blocks * 8, out, Xi, ivec, key);
+
+  if (CRYPTO_is_ARMv8_GCM_8x_capable() && len >= 192) {
```

Comment:
can we add a comment stating why `len` has to be `>=192`?

Reply:
Done.